book_source/adv_user_guide_web/Benchmarking.md

```diff
@@ -0,0 +1,81 @@
+# Benchmarking
```

Comment:
This file is generated on-the-fly from the Rmd version and thus shouldn't be committed.